README.md

```diff
@@ -64,6 +64,63 @@ and other docs.
 <br />
 <!-- END OF README-JOB SECTION -->
 
+## Homebound Fork
+
+This is Homebound's fork of `date-fns`. We forked this repository to extend the functionality of the `businessDate`
+functions. Our plan is to PR these improvements back to the mainstream project. However, in the meantime, we have
+logic to publish a `.patch` file to layer this functionality on top of the upstream `date-fns` package via the
+[`patch-package` project](https://github.com/ds300/patch-package).
+
+### Our Branch
+
+Currently, our code changes live on the [`improved-business-days`](https://github.com/homebound-team/date-fns/tree/improved-business-days)
+branch on our fork. The intention with this branch is that it's well-tested and will be PR'ed against the upstream
+project.
+
+This branch is tested using the same GitHub actions that run on the upstream project. When you push to this branch, it
+does not automatically trigger a new patch publish (see [below](#publishing-the-patch-package) for how to do that).
+
+### Publishing the Patch Package
+
+There are two distinct cases you might want to trigger a new package publish:
+
+1. If you've made changes on the `improved-business-days` branch and want to release them.
+1. If a new version of `date-fns` is released upstream.
+
+In either case, you trigger a release by running the [workflow in CircleCI](https://app.circleci.com/pipelines/github/homebound-team/date-fns?branch=homebound-patch-publish&filter=all). Just pick the last workflow run and click the "Rerun workflow from the start" button.
+
+![](https://imgur.com/XFnUtNc)
+
+The release process pulls the latest release tag, merges the `improved-business-days` branch and builds the TypeScript
+code. Then, it runs the `patch-package` process to produce a `.patch` file representing the changes.
+
+We publish the private NPM package, `@homebound/date-fns-patch` package for convenience. The versioning convention is
```

Comment:
thought: It might be interesting to make that patch public, and then see how many people use it. It could be a selling point for getting our PR merged in.

Reply:
Great idea! I thought more about this and, because there's no secret-sauce in that NPM package, I've made it public. See https://github.com/homebound-team/date-fns/pull/2/commits/9e9eac621199af2c272f5a9a3440db31a8424d61.